recipe/meta.yaml

```diff
@@ -38,7 +38,7 @@ outputs:
     build:
       number: {{ number }}
       script_env:
-        - CMAKE_GENERATOR = "Ninja"
+        - CMAKE_GENERATOR="Ninja"
```

Comment:
It appears that having spaces around the `=`s (while more readable) results in this environment variable not getting set to the value specified. So have dropped the spaces, which fixes the issue. Think this is an upstream bug in `conda-build` and/or `conda-mambabuild`, which needs follow up